src/coffee/helpers/user-agent.coffee

```diff
@@ -3,7 +3,7 @@ getPkg = require 'load-module-pkg'
 
 module.exports = (pkgName, pkgVersion) =>
   throw new Error 'Package name and version is required' unless pkgName and pkgVersion
-  sdkPkgVersion = getPkg('sphere-node-sdk').version
+  sdkPkgVersion = getPkg.sync('sphere-node-sdk').version
```

Comment:
Is this also related to some version update?

Reply:
Version `1.0.0` of the [load-module-pkg](https://www.npmjs.com/package/load-module-pkg#sync) expects a callback with the old API. I used the new synchronous API here.